app/models/article.rb

```diff
@@ -11,16 +11,26 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  account_id  :integer          not null
-#  author_id   :integer
+#  author_id   :bigint
 #  category_id :integer
 #  folder_id   :integer
 #  portal_id   :integer          not null
 #
+# Indexes
+#
+#  index_articles_on_author_id  (author_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#
 class Article < ApplicationRecord
+  include PgSearch::Model
+
   belongs_to :account
   belongs_to :category
   belongs_to :portal
-  belongs_to :folder
```

Comment:
Just when we finalize the whole flow, I will remove the respective table for this.